### DISCUSSION TITLE
Best Model for Context-Aware Recommendation

### DISCUSSION THREAD

**Discussion by steven-ja** [recommenders-team/recommenders#1858] [Q&A]
Hello Everyone,

In my project I am dealing with a music recommender with context data. An **example** can be found [here](https://github.com/irecsys/CARSKit/blob/master/context-aware_data_sets/Music_InCarMusic.zip)

I have already tried _LightFM, LightGBM Regressor_, however I am getting bad results. Hence my question is:

Which model of this repository would you recommend for **context data**?

Thank you in advance

**Comment by miguelgfierro** [comment]
100% LightGBM
I would try changing the features until you improve your results.
